src/sentry/static/sentry/app/components/issueDiff.jsx

```diff
@@ -0,0 +1,102 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+
+import ApiMixin from '../mixins/apiMixin';
+import LoadingIndicator from './loadingIndicator';
+import rawStacktraceContent from './events/interfaces/rawStacktraceContent';
+
+import '../../less/components/issueDiff.less';
+const IssueDiff = React.createClass({
+  propTypes: {
+    baseIssueId: PropTypes.string.isRequired,
+    targetIssueId: PropTypes.string.isRequired,
+    baseEventId: PropTypes.string.isRequired,
+    targetEventId: PropTypes.string.isRequired
+  },
+
+  mixins: [ApiMixin],
+
+  getDefaultProps() {
+    return {
+      baseEventId: 'latest',
+      targetEventId: 'latest'
+    };
+  },
+
+  getInitialState() {
+    return {
+      loading: true,
+      baseEvent: {},
+      targetEvent: {},
+      Diff: null
+    };
+  },
+
+  componentWillMount() {
+    let {baseIssueId, targetIssueId, baseEventId, targetEventId} = this.props;
+    Promise.all([
+      import('./splitDiff'),
+      this.fetchData(baseIssueId, baseEventId),
+      this.fetchData(targetIssueId, targetEventId)
+    ]).then(([{default: Diff}, baseEvent, targetEvent]) => {
+      this.setState({
+        Diff,
+        baseEvent: this.getException(baseEvent),
+        targetEvent: this.getException(targetEvent),
+        loading: false
+      });
+    });
+  },
+
+  getException(event) {
+    if (!event || !event.entries) return '';
+
+    const exc = event.entries.find(({type}) => type === 'exception');
```

Comment:
Just a heads up: there can be multiple exceptions (e.g. chained exceptions) for an event. This is most visible in Java and Ruby. How will that work here? Thought: we might literally need to page between exceptions stack traces.

I think it's okay if we punt on this though, as long as we open an issue to follow up.